modules/network-observability-flowcollector-configuration-parameters.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+
+// * configuring-operators.adoc
+
+:_content-type: REFERECNCE
+[id="network-observability-flowcollector-configuration-parameters{context}"]
+
+= Flow Collector Configuration Parameters
+The fields for the Flow Collector custom resource are described in the following table.
+
+.FlowCollector custom resource configuration parameters
```

Comment:
@skrthomas how do you envision the maintenance of this section in the long run? If I'm correct, this is not a pure copy-paste of the [upstream API reference](https://github.com/netobserv/network-observability-operator/blob/main/docs/FlowCollector.md) info. Would it be easier if we provide a full API reference is the same asciidoc format that would be just copy-pasted at each release? That's the intent for this PR: https://github.com/netobserv/network-observability-operator/pull/236

Reply:
I think this is a maintenance nightmare moving forward it would 💯 be better if we could somehow just auto-generate from the upstream reference.

Reply:
Incorporated into the documentation here: https://53263--docspreview.netlify.app/openshift-enterprise/latest/networking/network_observability/flowcollector-api.html, and there's a bunch of extraneous data that should not be populating above but below looks ok. Need to scrutinize further with the Vale linter but shouldn't be hard to figure out where that text is pulling in from.